.github/workflows/build.yml

```diff
@@ -124,7 +124,7 @@ jobs:
       - name: Cargo Test
         run: |
           make test-unit
-          make test-integration
+          make test-e2e
```

Comment:
Do we have a gap in test coverage if we don't run `make test-integration` along with these two others?  Or do they run now as part of `test-e2e`?

Reply:
They run as part of `test-e2e`. This make handle just also turns on the e2e-tests feature:
```
.PHONY: test-integration
test-integration:
	RUST_LOG=$(LOG_LEVEL) cargo test --test integration --no-fail-fast -- --nocapture --include-ignored

.PHONY: test-e2e
test-e2e:
	RUST_LOG=$(LOG_LEVEL) cargo test --test integration --features e2e-tests --no-fail-fast  -- integration_tests::test_dependencies --nocapture 
	RUST_LOG=$(LOG_LEVEL) cargo test --test integration --features e2e-tests --no-fail-fast -- --skip integration_tests::test_dependencies --nocapture --include-ignored
```

Reply:
Nice, thank you!